docs/reference/security/sts-login.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Secure Token Service (STS) Authentication for lakeFS
+description: Authenticate with lakeFS using Secure Token Service (STS) by leveraging a remote authenticator. This feature enables integration with Identity Providers (IdPs) for secure and efficient user authentication.
+grand_parent: Reference
+parent: Security
+redirect_from:
+  - /reference/remote-authenticator.html
+---
+
+# STS Login
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-cloud">lakeFS Cloud</a>
+{: .label .label-green }
+
+{: .d-inline-block }
+<a style="color: white;" href="#sso-for-lakefs-enterprise">lakeFS Enterprise</a>
+{: .label .label-purple }
+
+{: .note}
+> STS Login is available in lakeFS Cloud and lakeFS Enterprise. currently only tested on lakeFS Enterprise.
+
+Secure Token Service (STS) authentication in lakeFS allows users to authenticate via an Identity Provider (IdP), using OIDC Authentication workflow.
+This document outlines the process, from setting up the STS authentication flow to integrating with lakeFS through the [high-level Python SDK](../../integrations/python.md).
```

Comment:
```suggestion
This document outlines the process, from setting up the STS authentication flow to using temporary credentials to interact with lakeFS through the [high-level Python SDK](../../integrations/python.md).
```

Reply:
Temporary credentials is quite confusing, because it's both what they get from the IDP, but mostly what they get from us (the token) 
WDYT?

Reply:
Does short-lived token make more sense?

Reply:
it's about the same, it will be hard to differ them because they are basically the same thing, one provided by the Idp and one provided by us. let's use your suggestion, if it's confusing we will somehow change it